src/display/canvas.js

```diff
@@ -2112,8 +2112,10 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       var heightScale = Math.max(Math.sqrt(c * c + d * d), 1);
 
       var imgToPaint, tmpCanvas;
-      // instanceof HTMLElement does not work in jsdom node.js module
-      if (imgData instanceof HTMLElement || !imgData.data) {
+      // typeof check is needed due to node.js support, see issue #8489
+      if ((typeof HTMLElement === 'function' &&
+           imgData instanceof HTMLElement) ||
+           !imgData.data) {
```

Comment:
This line is indented with one space too many; let's re-format it like this instead:
```javascript
if ((typeof HTMLElement === 'function' &&
     imgData instanceof HTMLElement) || !imgData.data) {
```

Also, please squash the commits; refer to https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.